### ISSUE TITLE
website: android webview chromium version maybe incorrect or misleading

### ISSUE THREAD

**Issue by littlee** [Fyrd/caniuse#4988]
<img width="399" alt="WX20190704-111714@2x" src="https://user-images.githubusercontent.com/7534668/60637589-a6d29600-9e4d-11e9-8af5-5be1ec30e2a2.png">

I find that, Google Nexus 4 with Android 5.0 is using Chromium 37 in the system browser.

But the picture above is assuming or telling that Android 5.0+ device is using Chromium 67

Some reference links:

https://developer.chrome.com/multidevice/webview/overview

**Comment by jpmedley** [comment]
I agree this is misleading. One thing your link doesn't make clear is that the line, "WebView has moved to an APK so it can be updated separately to the Android platform" actually means WebView updates on a six week schedule using Chrome version numbers and largely having feature parity with Chrome. ([Here's a list](https://cs.chromium.org/chromium/src/android_webview/tools/system_webview_shell/test/data/webexposed/not-webview-exposed.txt) of Chrome features *not* supported in WebView from our testing code.)

37 is the first Chromium-based WebView to ship on an Android device, in Android 5.0. It should have auto-updated on the same schedule as Android. (If you've never connected the phone to wifi, it probably wouldn't. We try not to use up people's cellular data.)

The actual version of Android is not relevant after the release of Android 5.0. It would be if some time in the future we were to stop supporting Chrome on a specific release of Android.
